pkgs/by-name/us/usacloud/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+}:
+
+buildGoModule rec {
+  pname = "usacloud";
+  version = "1.14.1";
+
+  src = fetchFromGitHub {
+    owner = "sacloud";
+    repo = "usacloud";
+    tag = "v${version}";
+    hash = "sha256-bIVC0pa+54a/OpGKOM3LH7IMZnuYDvfGOMxcIwhHI7U=";
+  };
+
+  vendorHash = "sha256-m89iw5nk34KkCh8vlnKcwoPQXprUKzETYQNzi2CtX4c=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X=github.com/sacloud/usacloud/pkg/version.Revision=${src.rev}"
+  ];
+
+  doInstallCheck = true;
+  nativeInstallCheckInputs = [ versionCheckHook ];
+  versionCheckProgramArg = [ "--version" ];
+
+  meta = {
+    description = "CLI client for the Sakura Cloud";
+    homepage = "https://github.com/sacloud/usacloud";
```

Comment:
```suggestion
    homepage = "https://github.com/sacloud/usacloud";
    changelog = "https://github.com/sacloud/usacloud/releases/tag/v${version}/CHANGELOG.md"
```

Reply:
It seems there is no `CHANGELOG.md`, so I added `https://github.com/sacloud/usacloud/releases/tag/v${version}` instead.
Thank you!